view/code/index.ts

```diff
@@ -88,6 +88,21 @@ function listenChanges(input: HTMLInputElement): void {
     }
   }
 
+  function processRgbBlur(): void {
+    let value = input.value.trim()
+    let parsed = parseAnything(value)
+
+    if (parsed && parsed.mode in formats.get()) {
+      // Two options in <OutputFormats> — hex and rgb — are parsed as rgb.
+      let newFormat = isHexNotation(parsed.mode) ? 'hex' : parsed.mode
+      if (newFormat !== outputFormat.get()) {
+        outputFormat.set(newFormat as OutputFormats)
+      }
```

Comment:
Why we are not calling `setRgb` here?

Reply:
After the edits, this is not relevant. But initially the reason was that `setRgb` is also called inside `outputFormat.listen(..)`